### ISSUE TITLE
Hue: switching or not to the system channel types ?

### ISSUE THREAD

**Issue by lolodomo** [eclipse/smarthome#6037]
I studied a switch to the new system channel types for the hue binding.

One first difference I can see is that the brightness channel type is of category DimmableLight in the hue binding while Light in the system channel type.  My feeling is that it is an error when the system channel type was defined. Can someone confirm ?

The second difference is that the hue channel types have a "Lighting" tag. If we switch to the system channel types, this tag will be lost.

So do we switch to the system channel types or not ?

**Comment by lolodomo** [comment]
For information, I fixed the category for the system channel type in PR #6039 

The remaining question is now only the question about the tag.

**Comment by lolodomo** [comment]
I see a PR to add tags to the system channel types but should take time before it is merged.

What would be the consequence for the user of replacing a channel type with tag by a system channel type without tag ? Does it just mean that the user should add explicitely the tag in its items config file ?

**Comment by lolodomo** [comment]
@kaikreuzer @triller-telekom : go or no go ? I don't know the consequence of loosing the tag "Lighting" on the channel type.

**Comment by triller-telekom** [comment]
@lolodomo As you mentioned already, PR https://github.com/eclipse/smarthome/pull/5936 is still work in progress and we need to define the underlying tag library and query mechanism. Until we have defined that we will not add these tags. My PR was just meant as a starting point for the discussion to see how it looks in code.

If you replace a binding channel that has the tag "Lighting" with a system channel which does **not** have this tag, the tag will be gone unless the user specifies it explicitly in its configuration.

So I guess since we have not decided how exactly the tags will look like we should not change bindings that will loose their tags and thus unnecessarily break them now.

**Comment by kaikreuzer** [comment]
I agree, losing the current tags could be an unexpected change for users, so we should only do so once when rolling out the new tag library; so let us keep this change in the queue for the moment.

**Comment by lolodomo** [comment]
Ok, so system channel types were defined but they cannot be used ;)

**Comment by triller-telekom** [comment]
So far they can be used anywhere where we would not remove an existing tag :-)

**Comment by lolodomo** [comment]
Hue : only the color temperature channel could be replaced (no tag defined).
DMX: none could be replaced (always a tag defined).
Tradfri: color temperature and color channels could be replaced (no tag defined).
I am not sure that it is a good idea to do the change only partially.
